src/components/SearchForm/SearchForm.js

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 
-function SearchForm({ isMenuOpen }) {
+function SearchForm({ isMenuOpen, handleSearchArticles }) {
+  function handleSearchSubmit(e) {
+    e.preventDefault();
+    const keyword = e.currentTarget[0].value;
```

Comment:
`= e.currentTarget[0].value;`

this is a `pure`-js variant

You need to control inputs with `(`useState, value, onChange`)`


To read more about **controlled components**: https://reactjs.org/docs/forms.html#controlled-components